src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -100,7 +101,7 @@
       <_Dependency Remove="System.ValueTuple"/>
     -->
     <ItemGroup>
-      <_Dependency Remove="@(_Dependency)" Condition="$([MSBuild]::ValueOrDefault('%(Identity)', '').StartsWith('Microsoft.VisualStudio.'))"/>
+      <_Dependency Remove="@(_Dependency)" Condition="$([MSBuild]::ValueOrDefault('%(Identity)', '').StartsWith('Microsoft.VisualStudio.')) and %(Identity) != 'Microsoft.VisualStudio.LanguageServer.Protocol'"/>
```

Comment:
single quotes around `%(Identity)`

Reply:
Should we be inserting Protocol.Extensions as well?

Reply:
We do insert the dll, but don't need to update the AssemblyVersion.tt file - we use the same version variable for both (they are versioned together) and there isn't a version in the ngen search path that messes up ngen for that dll.  I'm not clear if this list is just to update the CSV though.

Reply:
Right, but this logic also inserts the package itself. See the artifacts:

![image](https://user-images.githubusercontent.com/41759/78952453-6d891f80-7a8a-11ea-87fa-b94018ee90ba.png)

How would that package be inserted otherwise?

Reply:
Interesting.  I'll dig into it because it's definitely getting inserted somehow right now, but perhaps incorrectly.

Reply:
Shouldn't these binaries be inserted only as part of VS.ExternalAPIs.Roslyn package and not as separate packages?